CookSmart/CookSmart/CSScaleView.m

```diff
@@ -196,7 +196,11 @@ static inline CGFloat unitsPerPoint(CSScaleView *scaleView)
 
 static inline CGFloat ptsPerUnit(CSScaleView *scaleView)
 {
-    return (SCALE_TILE_HEIGHT/scaleView.unitsPerTile);
+  CGFloat pts = (SCALE_TILE_HEIGHT/scaleView.unitsPerTile);
```

Comment:
Hmmm... that sounds like we have some kind of a problem with the view lifecycle. It sounds like this is being called when unitsPerTile is zero?